test/utils.py

```diff
@@ -179,20 +183,24 @@ def insertable_reduction(self, instrument: Instrument) -> Reduction:
         "964733aec28b00b13f32fb61afa363a74dd62130/mari/mari_mask2023_1.xml",
         "wbvan": 12345,
     },
-    reduction_state=ReductionState.NOT_STARTED,
+    state=State.NOT_STARTED,
+    owner=TEST_JOB_OWNER,
+    instrument=TEST_INSTRUMENT,
+    job_type=JobType.AUTOREDUCTION,
 )
 TEST_RUN = Run(
     instrument=TEST_INSTRUMENT,
     title="Whitebeam - vanadium - detector tests - vacuum bad - HT on not on all LAB",
-    experiment_number=1820497,
+    owner=TEST_JOB_OWNER,
     filename="MAR25581.nxs",
     run_start="2019-03-22T10:15:44",
     run_end="2019-03-22T10:18:26",
     raw_frames=8067,
     good_frames=6452,
     users="Wood,Guidi,Benedek,Mansson,Juranyi,Nocerino,Forslund,Matsubara",
-    reductions=[TEST_REDUCTION],
+    jobs=[TEST_JOB],
 )
+EXPERIMENT_NUMBERS = range(0, 100000)
```

Comment:
Is this value used?

Reply:
No I left it in by mistake another good catch, we used faker instead if you remember.